modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -52,6 +52,7 @@
     public static final String CONFIG_PACKAGE = "configPackage";
     public static final String BASE_PACKAGE = "basePackage";
     public static final String INTERFACE_ONLY = "interfaceOnly";
+    public static final String USE_SPRING_JPA = "useSpringJpa";
```

Comment:
@MarkusBansky I wonder if we could move this from the `SpringCodegen` up to the `AbstractJavaCodegen` so that it can be of benefit to ALL of the Java generators?